### ISSUE TITLE
Example does not work - __init__() got an unexpected keyword argument 'base_path'

### ISSUE THREAD

**Issue by salexln** [adobe/libLOL#1]
### Expected Behaviour
I should be able to run `lolc=LOLC(PlatformType.LINUX)`

### Actual Behaviour
I'm  getting the following error
```
TypeError                                 Traceback (most recent call last)
<ipython-input-9-219f8127ce29> in <module>
----> 1 lolc=LOLC(PlatformType.LINUX)

~/Library/Python/3.8/lib/python/site-packages/lol/api.py in __init__(self, platform, model_base)
     43             if model_base is None:
     44                 base_path = '{0}/bash_huge'.format(base_path)
---> 45             fte = LinuxFeatureExtraction(base_path=base_path)
     46             vectorizer_path = '{0}.vectorizer'.format(base_path)
     47             classifier_path = '{0}.classifier2'.format(base_path)

TypeError: __init__() got an unexpected keyword argument 'base_path'

#### Steps to Reproduce
```
1. Install pip: `pip install lolc --user`
2. Run `ipython`
3. Run
```
from lol.api import LOLC, PlatformType
lolc=LOLC(PlatformType.LINUX)
```

#### Platform and Version
Python 3.8.5
MacOS BigSur
lolc==0.1.0.0

**Comment by tiberiu44** [comment]
Hi @salexln ,

Thank you for letting us now. I confirm the bug. We will fix this ASAP.

**Comment by FredrikMS** [comment]
Also: 'data/win.bad.filtered' does not exist in the data directory trying to be read here: https://github.com/adobe/libLOL/blob/a70e5dfa657fcd693b3dff9767fde6273e97b151/lol/model/utils.py#L259

Not dure if this file was supposed to be included: https://github.com/adobe/libLOL/blob/a70e5dfa657fcd693b3dff9767fde6273e97b151/lol/data

**Comment by tiberiu44** [comment]
@salexln and @FredrikMS,

This should now be fixed with pip package version 0.1.0.1. I also pushed the updates on Github. There was a major issue with syncing our internal GIT with public. Sorry for this.

It will take some time for the change to propagate, but I think you can actually force this by doing `pip install lolc==0.1.0.1
`

Thanks for the help!

**Comment by tiberiu44** [comment]
Another fix is included in `lolc==0.1.0.2`, so please use this version.

**Comment by salexln** [comment]
@tiberiu44 thanks!!!
